apps/explorer/src/components/Layout/index.tsx

```diff
@@ -95,18 +95,22 @@ export function Layout() {
                             }}
                             toastOptions={{
                                 duration: 4000,
-                                success: {
-                                    icon: null,
-                                    className:
-                                        '!bg-success-light !text-success-dark',
-                                },
-                                error: {
-                                    icon: null,
-                                    className:
-                                        '!bg-issue-light !text-issue-dark',
-                                },
                             }}
-                        />
+                        >
+                            {(toast) => (
+                                <Banner
+                                    shadow
+                                    border
+                                    variant={
+                                        toast.type === 'success'
+                                            ? 'positive'
+                                            : 'error'
+                                    }
+                                >
+                                    {toast.message as string}
```

Comment:
From [the docs of react-hot-toast](https://react-hot-toast.com/docs/toaster) they suggest to use `resolveValue()`